var/spack/repos/builtin/packages/podio/package.py

```diff
@@ -29,6 +29,11 @@ class Podio(CMakePackage):
     version('0.9', sha256='3cde67556b6b76fd2d004adfaa3b3b6173a110c0c209792bfdb5f9353e21076f')
     version('0.8', sha256='9d035a7f5ebfae5279a17405003206853271af692f762e2bac8e73825f2af327')
 
+    variant('cxxstd',
+            default='17',
+            values=('17', '20'),
```

Comment:
Would this:
```suggestion
            values=('17', conditional('20', when='@0.14.1:')),
```
work?

Reply:
The issue here is with version:
```
>>> s = spack.spec.Spec('@0.14.1:')
>>> s.satisfies('@0.14')
True
```
The construct above should work if you e.g. use `0.13`

Reply:
Resolved with 002dceb0c3.